src/main/java/org/dependencytrack/model/Project.java

```diff
@@ -109,6 +111,7 @@
                 @Persistent(name = "parent")
         })
 })
+@Index(name = "PROJECT_NAME_IS_LATEST_IDX", members = {"name", "isLatest"})
```

Comment:
Do we need two indexes covering the `IS_LATEST` column?

Actually I believe we might need none. The DB query planner can already use the index on `NAME` to reduce the result to a specific project. Filtering the remaining records (i.e. all versions of that project) by `IS_LATEST` is then unlikely to yield much benefit by being indexed. But we pay a penalty for inserts and updates by having them.

Reply:
I removed it